guides/common/attributes.adoc

```diff
@@ -1,11 +1,11 @@
 // Document state: "nightly" for master, "stable" for last two releases,
 // "unsupported" for the rest and "satellite" for satellite build
-:DocState: nightly
+:DocState: unsupported
```

Comment:
Is this "unsupported" until 3.10 is GA? If so, then please ignore this comment.

Reply:
I believe so, but @ekohl might know better.